packages/core/src/tools/utils.ts

```diff
@@ -322,17 +322,44 @@ export interface EventEmitter {
   ): void
 }
 
+interface AddEventListenerOptions {
+  once?: boolean
+  capture?: boolean
+  passive?: boolean
+}
+
+/**
+ * Add an event listener to an event emitter object (Window, Element, mock object...).  This provides
+ * a few conveniences compared to using `element.addEventListener` directly:
+ *
+ * * supports IE11 by:
+ *   * using an option object only if needed
+ *   * emulating the `once` option
+ *
+ * * wraps the listener with a `monitor` function
+ *
+ * * returns a `stop` function to remove the listener
+ */
+export function addEventListener(
+  emitter: EventEmitter,
+  event: DOM_EVENT,
+  listener: (event: Event) => void,
+  options?: AddEventListenerOptions
+) {
+  return addEventListeners(emitter, [event], listener, options)
```

Comment:
nit: it could have been an opportunity to decouple wrapper + options logic from the logic that handle multiple events.

Reply:
There is a subtlety that would make it harder to do if the implementation was in `addEventListener`: using the `once` option makes sure the listener is called at most once, even when listening to  multiple events.

Reply:
oh, I would not expect that behavior.
Do we need it?

Reply:
Yes we'll need it. I'll add a documentation note about it.

Reply:
maybe it could be an extra option, like `onceForAll`

Reply:
In my mind, this is the expected behavior: the listener will be called at most once. I don't think we need both options: I can't imagine a use-case where we'd want to subscribe multiple events and the callback would be invoked once for each event.

Reply:
ok, let's go this way and we'll see how it goes